scout/export/panel.py

```diff
@@ -111,28 +113,30 @@ def export_panels(adapter, panels, versions=None, build="37"):
         yield gene_line
 
 
-def export_gene_panels(adapter, panels, version=None):
+def set_export_fields(panels: list[str]) -> list[tuple]:
+    """Set the field that should be exported for one panel or a list of panels."""
+
+    if len(panels) == 1:
+        return EXPORT_PANEL_FIELDS
+
+    # if more than one panel is exported (from the CLI) - export only HGNC and symbol
+    return EXPORT_PANEL_FIELDS[:2]
```

Comment:
..and that ensure the automation sees the same thing as before for almost every order. We still should test what happens if CG was to ask for a run with a single panel on a case. But as long as the first two columns stay the same, everything should be just fine.